Event_spec.ts

```diff
@@ -98,6 +102,40 @@ type GLN = string
  */
 type GLNExtension = string
 
+/**
+* A Position is an array of (lon, lat) coordinates.
+* https://tools.ietf.org/html/rfc7946#section-3.1.1
+*/
+
+type GeoPosition = number[]
+
+/**
+* A GeoJSON geometry object.
+* https://tools.ietf.org/html/rfc7946#section-3
+*/
+interface IGeometry {
```

Comment:
The Idiomatic Typescript definition is `type Geometry = IPoint | IPolygon`

See https://www.typescriptlang.org/docs/handbook/advanced-types.html section "Discriminated Unions"